src/mam4xx/mo_photo.hpp

```diff
@@ -0,0 +1,552 @@
+#ifndef MAM4XX_MO_PHOTO_HPP
+#define MAM4XX_MO_PHOTO_HPP
+
+#include <haero/math.hpp>
+#include <mam4xx/mam4_types.hpp>
+#include <mam4xx/utils.hpp>
+
+namespace mam4 {
+
+namespace mo_photo {
+
+// number of vertical levels
+constexpr int pver = 72;
```

Comment:
@pressel and I were discussing the fact that we seem to be hard-wiring the number of vertical levels in given configurations. To be honest, this isn't a huge deal right now as long as we make it possible to configure this parameter for mam4xx at compile time, because it's also [hardwired in EAMxx](https://github.com/eagles-project/scream/blob/master/components/eamxx/src/scream_config.h.in#L20). But it's something that we should be aware of.

To make sure we don't forget about this, I'll create two issues:

1. To provide a CMake parameter for the number of vertical levels so that we can configure it in the same way as it is in EAMxx (for the short term)
2. A reminder for us to revisit this and see how we might (if practical) convert this to a run-time parameter.

Reply:
See #223 and #224